src/cdk-experimental/listbox/listbox.ts

```diff
@@ -46,6 +56,39 @@ let nextId = 0;
 //   - should listbox wrap be configurable?
 //   - should skipping disabled options be configurable?
 
+/**
```

Comment:
* I think we should let people enable/disable wrapping
* I think it's probably easiest for us to support both ways for disabled options. 

I'm iffy on giving direct access to the keymanager, though, in part because I'd like the freedom to change the key manager implementation under the hood in the future. It seems like it would be mostly straightforward to support these as inputs + default options?

Reply:
It looks like `ActiveDescendantKeyManager` doesn't even support stopping on disabled items currently, so I'll leave that as a future feature to add if people request it. And for the wrapping I'll add an input for it

Reply:
You should be able to allow stopping on disabled items using `skipPredicate(() => false)`.

Reply:
Ah, I missed that. I was looking for something with the word disabled in it

Reply:
Added these in the latest commit, PTAL